core/loadpoint_test.go

```diff
@@ -526,13 +529,25 @@ func cacheExpecter(t *testing.T, lp *Loadpoint) (*util.Cache, func(key string, v
 	go cache.Run(paramC)
 
 	expect := func(key string, val interface{}) {
-		p := cache.Get(key)
-		t.Logf("%s: %.f", key, p.Val) // REMOVE
-		if p.Val != val {
-			t.Errorf("%s wanted: %.0f, got %v", key, val, p.Val)
+		// wrap in retry, as cache async update otherwise causes flaky tests
+		err := retry.Do(
```

Comment:
Ich versteh nur Bahnhof. Evtl. sollten wir lieber die Tests korigieren.

Reply:
Der Test ist OK. Das ist mehr in prinzipielles Problem. Der Cache hier hängt auf dem Bus, und liest asynchron die gepublishten Updates (z.B. für den Guard Timer) mit. Je nach Timing der Go Routine für den Cache kann es passieren, daß der Cache einen geänderten Wert noch nicht aus dem Channel gelesen hat während der eigentliche Test Case bereits diesen erwartet. Der Retry hier sorgt einfach dafür, daß wir nicht direkt beim ersten Mismatch (alter Wert) aufgeben, sondern es noch mal probieren. Die 100ms die ein Retry wartet, geben bei meinen lokalen Tests der zweiten Go Routine genug Zeit um den Cache zu aktualisieren.

Reply:
>Die 100ms die ein Retry wartet, geben bei meinen lokalen Tests der zweiten Go Routine genug Zeit um den Cache zu aktualisieren.

Die 100ms waren ja nicht schön- aber warum können wir die nicht so lassen wenn sie funktionieren?